packages/fx-core/resource/package.nls.json

```diff
@@ -577,6 +581,18 @@
   "core.uninstallQuestion.tdpOption": "App registration",
   "core.uninstallQuestion.botOption": "Bot framework registration",
   "core.uninstallQuestion.projectPath": "Project path",
+  "core.syncManifest.projectPath": "Project path",
+  "core.syncManifest.env": "Target Teams Toolkit Environment",
+  "core.syncManifest.teamsAppId": "Teams App ID (optional)",
+  "core.syncManifest.addWarning": "There are new properties added to the manifest template, please manually update the local manifest. Diff Path: %s. New Value %s.",
```

Comment:
New properties added to the manifest template. Manually update the local manifest. Diff Path: %s. New Value %s.

Reply:
For this warning, users need to update the manifest by their own. What about adding a "Please" before the "Manually update the local manifest"?